examples/dagcbor.rs

```diff
@@ -0,0 +1,53 @@
+use serde::de::{Deserialize, Deserializer};
+use serde::ser::{Serialize, Serializer};
+use serde_cbor::{DeserializerExt, SerializerExt, Value};
+use std::error::Error;
+use std::fs::File;
+use std::io::Cursor;
+
+#[derive(Debug, PartialEq)]
+struct Cid(Vec<u8>);
```

Comment:
What are "dagcbor" a "Cid"?

Reply:
This is just a test. I need tags support for IPFS/IPLD, just like @vmx and @dignifiedquire... https://docs.ipfs.io/guides/concepts/cid/

Can change this to another example if you want.

Reply:
> Can change this to another example if you want.

Maybe this is easier than explaining the example for people who don't know IPFS. For example show how to decode this message from a "bookmark service". It has tagged values for dates and URLs. 

```
{
"title": "The Example Domain",
"link": 32("http://example.org/"),
"created": 0("2003-12-13T18:30:02Z")
}
```